<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QToolBox" name="mainLayoutToolbox">
      <widget class="QWidget" name="dataLoadingPage" native="true">
       <attribute name="label">
        <string>Data Loading</string>
       </attribute>
       <layout class="QGridLayout" name="mainLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="dataLoadingLayout">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>1072</width>
            <height>150</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="dataset1Layout">
             <item>
              <widget class="QLabel" name="dataset1Label">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Dataset A</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dataset1FileLoad">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Select File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="dataset1Columns">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="dataset1ProgressBar">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="dataset2Layout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="dataset2Label">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Dataset B</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dataset2FileLoad">
               <property name="text">
                <string>Select File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="dataset2Columns">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="dataset2ProgressBar">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <widget class="Line" name="addremoveLineLeft">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <layout class="QVBoxLayout" name="dataButtonLayout">
             <item>
              <widget class="QPushButton" name="addOneButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addAllButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeOneButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeAllButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;&lt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="4">
            <widget class="Line" name="addremoveLineRight">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <layout class="QVBoxLayout" name="comparisonColumnsLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="comparisonColumnsLabel">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Comparisons</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="comparisonColumnsTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataDetailPage">
       <attribute name="label">
        <string>Data Detail</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QFrame" name="dataDetailLayout">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTabWidget" name="datasetDetailsTabLayout">
             <property name="tabShape">
              <enum>QTabWidget::Triangular</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="dataset1Tab">
              <attribute name="title">
               <string>Dataset 1</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <widget class="QTableWidget" name="dataframe1Table"/>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="datasetDetails1Button">
                 <property name="text">
                  <string>Dataset Details</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dataset2Tab">
              <attribute name="title">
               <string>Dataset 2</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0">
                <widget class="QTableWidget" name="dataframe2Table"/>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="datasetDetails2Button">
                 <property name="text">
                  <string>Dataset Details</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTabWidget" name="comparisonsTabLayout">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="tabShape">
              <enum>QTabWidget::Triangular</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="setupTab">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <attribute name="title">
               <string>Setup</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <layout class="QGridLayout" name="comparisonSetupParentLayout">
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="finalButtonsLayout">
                   <item>
                    <widget class="QPushButton" name="compareButton">
                     <property name="text">
                      <string>Compare</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="resetButton">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>Reset</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="comparisonSetupLayout">
                   <item>
                    <layout class="QVBoxLayout" name="setupInputsLayout">
                     <item alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="comparisonSetupLabel">
                       <property name="text">
                        <string>Comparison Setup:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="datasetsComboBox"/>
                     </item>
                     <item>
                      <widget class="QComboBox" name="comparisonsCheckbox"/>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="addDiffCheckbox">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Add Difference Column</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="performValidationsCheckbox">
                       <property name="text">
                        <string>Perform Validations</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="createVizCheckbox">
                       <property name="text">
                        <string>Create Visualizations</string>
                       </property>
                      </widget>
                     </item>
                     <item alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="inputParamtersLabel">
                       <property name="text">
                        <string>Input Parameters:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTextEdit" name="inputParametersTextEdit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="setupConfigLayout">
                     <item alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="validationConfiguration">
                       <property name="text">
                        <string>Validation Configuration:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="openConfigButton">
                       <property name="text">
                        <string>Open Config File</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTableView" name="configTable"/>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="addRemoveButtonLayout">
                       <item>
                        <widget class="QPushButton" name="addValidationButton">
                         <property name="text">
                          <string>+</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="removeValidationButton">
                         <property name="enabled">
                          <bool>false</bool>
                         </property>
                         <property name="text">
                          <string>-</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="comparisonTab">
              <attribute name="title">
               <string>Comparisons</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="comparisonLayout">
                   <item alignment="Qt::AlignHCenter">
                    <widget class="QLabel" name="comparisonLabel">
                     <property name="text">
                      <string>Comparison</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTableView" name="comparisonTable"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="validationLayout">
                   <item alignment="Qt::AlignHCenter">
                    <widget class="QLabel" name="validationsLabel">
                     <property name="text">
                      <string>Validations</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTableView" name="validationTable"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="chartsTab">
              <attribute name="title">
               <string>Charts</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="chartsLayout">
                 <item>
                  <widget class="QComboBox" name="chartComboBox"/>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="chartsGraphics"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
